src/components/menu-mobile/Menu.vue

```diff
@@ -1,40 +1,129 @@
 <template>
-  <div class="mew-component--wallet-menu-mobile">
-    <v-overlay :value="value" opacity="0.95">
-      <div style="overflow-y: auto">
-        <v-btn class="exit" x-large icon @click="$emit('input')">
-          <v-icon>mdi-window-close</v-icon>
-        </v-btn>
+  <div class="mew-component--landing-page-menu-mobile">
+    <mew-button
+      class="mobile-menu-button ml-n3"
+      color-theme="white"
+      btn-style="transparent"
+      icon="mdi-text"
+      icon-type="mdi"
+      icon-align="left"
+      style="border-radius: 100% !important; padding: 0"
+      @click.native="isOpen = true"
+    />
 
-        <ul class="menu-content">
-          <li v-for="(m, k) in menu" :key="k">
-            <div class="menu-divider"></div>
+    <v-navigation-drawer
+      v-model="isOpen"
+      height="100%"
+      width="100%"
+      absolute
+      temporary
+      color="expandHeader"
+    >
+      <v-card flat class="px-2" color="expandHeader">
+        <v-toolbar flat color="expandHeader" dark class="pt-7">
+          <v-btn class="exit mr-n12" x-large icon @click="isOpen = false">
```

Comment:
Can you make this button on the right please

Reply:
Fixed